superbit_lensing/galsim/mock_superBIT_data.py

```diff
@@ -799,21 +884,38 @@ def main():
         full_image = galsim.ImageF(sbparams.image_xsize, sbparams.image_ysize)
         sky_level = sbparams.exp_time * sbparams.sky_bkg
         full_image.fill(sky_level)
-        full_image.setOrigin(0,0)
-
-        # If you wanted to make a non-trivial WCS system, could set theta to a non-zero number
-        theta = 0.0 * galsim.degrees
-        dudx = np.cos(theta) * sbparams.pixel_scale
-        dudy = -np.sin(theta) * sbparams.pixel_scale
-        dvdx = np.sin(theta) * sbparams.pixel_scale
-        dvdy = np.cos(theta) * sbparams.pixel_scale
-        image_center = full_image.true_center
-        affine = galsim.AffineTransform(dudx, dudy, dvdx, dvdy, origin=full_image.true_center)
-        sky_center = galsim.CelestialCoord(ra=sbparams.center_ra, dec=sbparams.center_dec)
-
-        wcs = galsim.TanWCS(affine, sky_center, units=galsim.arcsec)
+
+        ## Define X & Y dither offsets
+        dither_offsets = rng.integers(-100, 100, size=2)
+        full_image.setOrigin(dither_offsets[0], dither_offsets[1])
         full_image.wcs = wcs
 
+
+        ##
```

Comment:
This code block can be deleted now that everything is wrapped up in `_set_seeds()`

Reply:
That's true. We can remove it during the pull into `master`.